src/App.tsx

```diff
@@ -1,26 +1,154 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { FormEvent, useMemo, useState, useEffect } from 'react';
+import cn from 'classnames';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { getTodos } from './api/todos';
+import { FooterPart } from './components/Footer';
+import { ErrorMessage } from './components/errorsMessage';
+import { Status } from './components/status';
+import { createTodos, USER_ID } from './api/todos';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  const [todoTemplate, setTodoTempalte] = useState<Todo | null>(null);
+  const [error, setError] = useState<ErrorMessage | ''>('');
+  const [status, setStatus] = useState<Status>(Status.All);
+  const [valueTitle, setValue] = useState<string>('');
+  const [disableInput, setDisableInput] = useState(false);
+
+  const uploadingTodos = useMemo(() => {
+    setError(ErrorMessage.noError);
+
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setError(ErrorMessage.loadError);
+        setTimeout(() => {
+          setError(ErrorMessage.noError);
+        }, 3000);
+      })
+      .finally();
+  }, []);
+
+  function deleteError() {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
  const deleteError = () => {
```